README.md

```diff
@@ -1,5 +1,20 @@
 ![We Got Coders Logo](http://wegotcoders.com/assets/logos/WEGOTCODERS_color.png "We Got Coders")
 
+
+#Some notes on my programming challenge
+
+I wasn't sure quite how to deal with this app that you sent me, I found working on the individual languages in a console window fairly easy, but this totally threw me.
+
+As I wasn't sure how to do a lot of this properly, so I decided to do what I could, and therefore demonstrate some of my programming knowlege.  
+
+For example I'm not sure where the content I was supposed to be laying out in main.erb was located, or how to arrange it if I couldn't see it. So I created a new page with my profile information on it.
```

Comment:
In the real world we'll be on hand to ask; in this case I'm looking for your thought process and whether you can use your initiative. I'll take a look and let you know what I think!